gensim/parsing/porter.py

```diff
@@ -98,39 +132,147 @@ def _m(self):
             i += 1
 
     def _vowelinstem(self):
-        """True <=> 0,...j contains a vowel"""
+        """Check if b[i] (i = 0, ... , j) contains a vowel letter.
+
+        Returns
+        -------
+        bool
+
+        Examples
+        --------
+        >>> from gensim.parsing.porter import PorterStemmer
+        >>> p = PorterStemmer()
+        >>> p.b = "gnsm"
+        >>> p.j = 3
+        >>> p._vowelinstem()
+        False
+
+        >>> from gensim.parsing.porter import PorterStemmer
+        >>> p = PorterStemmer()
+        >>> p.b = "gensim"
+        >>> p.j = 5
+        >>> p._vowelinstem()
+        True
+
+        """
         return not all(self._cons(i) for i in xrange(self.j + 1))
 
     def _doublec(self, j):
-        """True <=> j,(j-1) contain a double consonant."""
+        """Check if b[j], b[j - 1] contain a double consonant letter.
+
+        Parameters
+        ----------
+        j : int
+            Index for `b`
+
+        Returns
+        -------
+        bool
+
+        Examples
+        --------
+        >>> from gensim.parsing.porter import PorterStemmer
+        >>> p = PorterStemmer()
+        >>> p.b = "real"
+        >>> p.j = 3
+        >>> p._doublec(3)
+        False
+
+        >>> from gensim.parsing.porter import PorterStemmer
+        >>> p = PorterStemmer()
+        >>> p.b = "really"
+        >>> p.j = 5
+        >>> p._doublec(4)
+        True
+
+        """
         return j > 0 and self.b[j] == self.b[j - 1] and self._cons(j)
 
     def _cvc(self, i):
-        """True <=> i-2,i-1,i has the form consonant - vowel - consonant
-        and also if the second c is not w,x or y. This is used when trying to
-        restore an e at the end of a short word, e.g.
+        """Check if b[i - 2], b[i - 1], b[i] have the form consonant letter - vowel letter- consonant letter
```

Comment:
"have the form consonant letter - vowel letter- consonant letter" - > "make the (consonant, vowel, consonant) pattern"